core/iwasm/aot/aot_intrinsic.c

```diff
@@ -419,10 +421,15 @@ aot_intrinsic_f64_to_f32(float64 f)
     return (float32)f;
 }
 
+/* The epsilon value is from https://www.cplusplus.com/reference/cfloat/ */
+
 int32
 aot_intrinsic_f32_cmp(AOTFloatCond cond, float32 lhs, float32 rhs)
 {
     switch (cond) {
+        case FLOAT_EQ:
+            return fabsf(lhs - rhs) <= 1e-5f ? 1 : 0;
```

Comment:
fabsf isn't supported in zephyr platform, whose math lib is ported from open source library, see core/shared/platform/common/math/math.c. Can we use `(float32)fabs(lhs - rhs)` instead?
Another issue is that do we need to check <= 1e-5f? In interpreter f32.cmp opcode is just interpreted with f1 == f2.

Reply:
It's better to check it.